.github/workflows/check.yml

```diff
@@ -10,8 +10,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      # Use the jq tool (included in GitHub runner environments) to simplistically validate JSON files.
+      # Using git grep to fail the build if any tab characters are found in source files - we use spaces for whitespace.
+      # Using the jq tool (included in GitHub runner environments) to simplistically validate JSON files.
       - run: |
+          ! git --no-pager grep $'\t'
```

Comment:
I wonder if it's worth outputting something like:

"Lint failed! Source files must use spaces for whitespace, but the following files use tabs: `<insert list>`"

Otherwise I feel the build failure would be pretty cryptic?

Reply:
Hopefully this addresses your concern (albeit not exactly the same form you suggest): https://github.com/ably/ably-common/pull/78/commits/3e48a5664917cc9b5d57453c2b5db6b0fd7b1a6e

Reply:
That'll do 🙂